teos/teosd.py

```diff
@@ -218,9 +221,8 @@ def bootstrap_components(self):
             elif len(missed_blocks_responder) != 0 and len(missed_blocks_watcher) != 0:
                 Builder.update_states(self.watcher, missed_blocks_watcher, missed_blocks_responder)
 
-        # Fire ChainMonitor
-        # FIXME: 92-block-data-during-bootstrap-db
-        self.chain_monitor.monitor_chain()
+        # Activate ChainMonitor
+        self.chain_monitor.activate()
```

Comment:
As commented in chat, I planned `start_services` more as bringing up all the interfaces, while `bootstrap_components` for starting everything internal of the watchtower. So leaving the way it is for now − it might need to be further refactored at some point to make the TeosDaemon testable, I suppose, but leaving that for future PRs.